python_modules/libraries/dagster-k8s/dagster_k8s_tests/unit_tests/test_executor.py

```diff
@@ -119,6 +139,8 @@ def test_step_handler_user_defined_config(kubeconfig_file):
         assert kwargs["body"].spec.template.spec.containers[0].image == "bizbuz"
         job_resources = kwargs["body"].spec.template.spec.containers[0].resources
         assert job_resources.to_dict() == expected_resources
+        assert kwargs["body"].spec.template.spec.containers[0].env[1].name == "FOO_TEST"
```

Comment:
a test case that verifies that they are added and not replaced would be cool (one env var from the run launcher, one from the executor, combined into two total)